### DISCUSSION TITLE
incompatibilities simply not working

### DISCUSSION THREAD

**Discussion by CryptoCockseth** [nftchef/art-engine#217] [Help + Q&A]
I have a bunch of incompatibilities, they all work, except a few. 

The example I have is, if short hair, then no bucket, written like "Short Hair": ["Bucket"] yet short hair still appears with the bucket, and yes, hair comes first in the layer order. So there is no reason this shouldnt work. both are perfect CleanNames and yet i cannot find the solution.  multiple Short Hairs are still existing with the bucket even though they should not. 

Please help. 

Thank you.

**Comment by alla0242** [comment]
what's been working for me is:

Red: ["Short Hair"],
"Short Hair": ["Bucket"],
White: ["_ALL THE ITEMS TO CHOOSE INSTEAD_", "_IF YOU NEED MORE THAN ONE ITEM_"],


Hope this works for ya

**Comment by cryptofannatic** [comment]
@CryptoCockseth did you manage to figure out a solution? I'm facing the same issue and I've spent countless hours but unable to figure this out.
